mcnp_input_reader.py

```diff
@@ -223,29 +438,185 @@ def cells_with_mat(mat_num, cell_list):
     return cells
 
 
-def print_cell(cell):
-    """ pretty printing of cell object """
-    print("Cell number: ", cell.number)
-    print("Cell material: ", cell.mat)
-    print("Cell density: ", cell.density)
-    print("Cell geom: ", cell.geom)
-    print("Cell surfaces: ", cell.surfaces)
-    print("Cell imp p:", cell.imp_p)
-    print("Cell imp n:", cell.imp_n)
+def float_check(n):
+    """ tries to take input as float, returns false if error """
+    try:
+        float(n)
+        return True
+    except Exception:
+        return False
+
+
+def surface_reader(fpath):
+    """ reads from the mcnp file and outputs
+    dataframe with the details of the surfaces """
+    ifile = ut.get_lines(fpath)
+    cell_bloc, surf_bloc, data_bloc = split_blocs(ifile)
+    # takes the blank and c line out
+    del surf_bloc[0:2]
+
+    # Assumes that a surface description that goes over one line
+    # will have a whitespace at the end of every continuing line.
+    bloc = []
+    comments_list = []
+    for i, row in enumerate(surf_bloc):
+        if row[-1] == ' ':
+            surf_bloc[i] = surf_bloc[i] + surf_bloc[i+1]
+            surf_bloc.pop(i+1)

```

Comment:
changing the size of a list whilst looping over it is generally not good practice